cdap-app-fabric/src/main/java/co/cask/cdap/metadata/LineageAdmin.java

```diff
@@ -117,11 +127,26 @@ public boolean apply(Relation relation) {
    * @param startMillis start time period
    * @param endMillis end time period
    * @param levels number of levels to compute lineage for
+   * @param rollup lineage collapse rollup
```

Comment:
Maybe something like - `indicates whether to aggregate programs, currently supports rolling up programs into workflows`